tests/integration/docusaurus/tutorials/getting-started/getting_started.py

```diff
@@ -84,7 +84,6 @@
 # NOTE: The following assertion is only for testing and can be ignored by users.
 assert isinstance(validator, Validator)
 
-# <snippet name="tests/integration/docusaurus/tutorials/getting-started/getting_started.py exclude_column_names">
```

Comment:
Good catch! I guess this one isn't used anywhere now?

Reply:
Yup :)